<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sloth</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/sloth.svg</normaloff>:/icons/sloth.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_Image"/>
   </widget>
   <widget class="QMenu" name="menu_Views">
    <property name="title">
     <string>&amp;Views</string>
    </property>
    <addaction name="actionLocked"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuShortcuts">
    <property name="title">
     <string>&amp;Shortcuts</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnnotation">
    <property name="title">
     <string>&amp;Annotation</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuShortcuts"/>
   <addaction name="menu_Views"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAnnotation"/>
   <addaction name="menuPlugins"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="action_Add_Image"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockProperties">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>44</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockPropertiesContents"/>
  </widget>
  <widget class="QDockWidget" name="dockAnnotations">
   <property name="windowTitle">
    <string>Annotations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockAnnotationsContents"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder_table.png</normaloff>:/icons/folder_table.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open existing annotation file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/page_white_text.png</normaloff>:/icons/page_white_text.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create new annotation file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/door_in.png</normaloff>:/icons/door_in.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="action_Add_Image">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/picture_add.png</normaloff>:/icons/picture_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/arrow_right.png</normaloff>:/icons/arrow_right.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/arrow_left.png</normaloff>:/icons/arrow_left.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionMask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mask</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/disk_multiple.png</normaloff>:/icons/disk_multiple.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Point</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionLocked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Locked</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
